docs/METRICS.md

```diff
@@ -9,8 +9,9 @@ grabbit exposes and reports the following metrics to Prometheus
 | grabbit       | handlers      | result                            | records and counts each run of a handler, having the handler's name, message type and the result as labels|
 | grabbit       | handlers      | latency                           | records the execution time of each run of a handler, having the handler's name, message type as labels|
 | grabbit       | messages      | rejected_messages                 | increments each time a message gets rejected                                |
-| grabbit       | saga          | timedout_sagas                   | counting the number of timedout saga instances                              |
-| grabbit       | outbox        | outbox_total_records                   | reports the total amount of records currently in the outbox                              |
-| grabbit       | outbox        | outbox_pending_delivery                   | reports the total amount of records pending delivery currently in the outbox                              |
-| grabbit       | outbox        | outbox_pending_removal                   | reports the total amount of records that were sent and pending removal currently in the outbox                              |
+| grabbit       | saga          | timedout_sagas                    | counting the number of timedout saga instances                              |
+| grabbit       | saga          | latency                           | records the execution time of the entire saga, from it's creation until deletion. having the "Service", "SagaType" as labels|
```

Comment:
@kerenbe4 it is worth mentioning in what units are we reporting this metric (milliseconds)